packages/core/src/provider/schema.ts

```diff
@@ -0,0 +1,66 @@
+import { Address as zAddress } from "abitype/zod";
+import type { Chain, Transport } from "viem";
+import z from "zod";
+import type { PublicErc4337Client, SupportedTransports } from "../client/types";
+import { getChain } from "../utils/index.js";
+
+export const SmartAccountProviderOptsSchema = z.object({
+  /**
+   * The maximum number of times to try fetching a transaction receipt before giving up (default: 5)
+   */
+  txMaxRetries: z.number().optional().default(5),
```

Comment:
ah for the `infer` vs `output` vs `input` i meant it in regard to issues you ran into with defaults? are you still having issues with defaults?

Reply:
yeah still had issues. can run it by you next week